swarm.cabal

```diff
@@ -147,7 +147,7 @@ library
     build-depends:    base                          >= 4.14 && < 4.18,
                       aeson                         >= 2 && < 2.2,
                       array                         >= 0.5.4 && < 0.6,
-                      brick                         >= 1.0 && < 1.5,
+                      brick                         >= 1.0 && < 1.6,
```

Comment:
Should this be `brick >= 1.5 && < 1.6`?  I assume it will not compile with earlier versions of `brick`, right?

Reply:
correct